nemo/collections/asr/parts/submodules/conformer_modules.py

```diff
@@ -50,6 +63,9 @@ def __init__(
         d_model,
         d_ff,
         self_attention_model='rel_pos',
+        global_tokens=0,
+        global_tokens_placing="start",
+        global_attn_separate=False,
```

Comment:
Does enabling global_attn_separate helps?

Reply:
In my tests results are pretty similar